src/images/colormaps/index.js

```diff
@@ -50,6 +52,8 @@ const colormaps = {
   Thermal: { name: "thermal", path: thermal },
   Topo: { name: "topo", path: topo },
   Turbid: { name: "turbid", path: turbid },
+  Red_scale: { name: "red_scale", path: red_scale },
+  Blue_scale: { name: "blue_scale", path: blue_scale },
```

Comment:
I would just leave as red and blue because the names aren't taken yet. Also, could you put them in alphabetical order for completions sake?
```suggestion
  Red: { name: "red", path: red_scale },
  Blue: { name: "blue", path: blue_scale },
```

Reply:
I think `Red` will imply that is only a hard a color. I used the postfix `scale` indicating that there some kind of transition. I'm open to use another prefix/postfix.

Reply:
Okay sounds good! I would name them as `RedScale` and `BlueScale` to keep the naming scheme then

Reply:
Done.